worlds/yoshisisland/Items.py

```diff
@@ -0,0 +1,122 @@
+    'World 4 Gate': ItemData('Gates', 0x302071, ItemClassification.progression),
+    'World 5 Gate': ItemData('Gates', 0x302072, ItemClassification.progression),
+    'World 6 Gate': ItemData('Gates', 0x302073, ItemClassification.progression),
+
+    'Extra 1': ItemData('Panels', 0x302074, ItemClassification.progression),
+    'Extra 2': ItemData('Panels', 0x302075, ItemClassification.progression),
+    'Extra 3': ItemData('Panels', 0x302076, ItemClassification.progression),
+    'Extra 4': ItemData('Panels', 0x302077, ItemClassification.progression),
+    'Extra 5': ItemData('Panels', 0x302078, ItemClassification.progression),
+    'Extra 6': ItemData('Panels', 0x302079, ItemClassification.progression),
+    'Extra Panels': ItemData('Panels', 0x30207A, ItemClassification.progression),
+
+    'Bonus 1': ItemData('Panels', 0x30207B, ItemClassification.progression),
+    'Bonus 2': ItemData('Panels', 0x30207C, ItemClassification.progression),
+    'Bonus 3': ItemData('Panels', 0x30207D, ItemClassification.progression),
+    'Bonus 4': ItemData('Panels', 0x30207E, ItemClassification.progression),
+    'Bonus 5': ItemData('Panels', 0x30207F, ItemClassification.progression),
+    'Bonus 6': ItemData('Panels', 0x302080, ItemClassification.progression),
+    'Bonus Panels': ItemData('Panels', 0x302082, ItemClassification.progression),
+
+    'Anytime Egg': ItemData('Consumable', 0x302083, ItemClassification.useful, 0),
+    'Anywhere Pow': ItemData('Consumable', 0x302084, ItemClassification.filler, 0),
+    'Winged Cloud Maker': ItemData('Consumable', 0x302085, ItemClassification.filler, 0),
+    'Pocket Melon': ItemData('Consumable', 0x302086, ItemClassification.filler, 0),
+    'Pocket Fire Melon': ItemData('Consumable', 0x302087, ItemClassification.filler, 0),
+    'Pocket Ice Melon': ItemData('Consumable', 0x302088, ItemClassification.filler, 0),
+    'Magnifying Glass': ItemData('Consumable', 0x302089, ItemClassification.filler, 0),
+    '+10 Stars': ItemData('Consumable', 0x30208A, ItemClassification.useful, 0),
+    '+20 Stars': ItemData('Consumable', 0x30208B, ItemClassification.useful, 0),
+    '1-Up': ItemData('Lives', 0x30208C, ItemClassification.filler, 0),
+    '2-Up': ItemData('Lives', 0x30208D, ItemClassification.filler, 0),
+    '3-Up': ItemData('Lives', 0x30208E, ItemClassification.filler, 0),
+    '10-Up': ItemData('Lives', 0x30208F, ItemClassification.filler, 5),
+    'Bonus Consumables': ItemData('Events', None, ItemClassification.progression, 0),
+    'Bandit Consumables': ItemData('Events', None, ItemClassification.progression, 0),
+    'Bandit Watermelons': ItemData('Events', None, ItemClassification.progression, 0),
+
+    'Fuzzy Trap': ItemData('Traps', 0x302090, "trap", 0),
+    'Reversal Trap': ItemData('Traps', 0x302091, "trap", 0),
+    'Darkness Trap': ItemData('Traps', 0x302092, "trap", 0),
+    'Freeze Trap': ItemData('Traps', 0x302093, "trap", 0),

```

Comment:
This crashes if traps are enabled in the yaml, btw. It needs to be `ItemClassification.trap` instead of "trap".

Reply:
NO I DIDN'T MEAN TO APPROVE LOL